snuba/query/processors/granularity_processor.py

```diff
@@ -48,11 +48,11 @@ class GranularityMapping(NamedTuple):
     enum_value: int
 
 
-PERFORMANCE_GRANULARITIES: Sequence[GranularityMapping] = [
-    GranularityMapping(60, 1),
-    GranularityMapping(3600, 2),
-    GranularityMapping(86400, 3),
-]
+PERFORMANCE_GRANULARITIES: Mapping[int, int] = {
```

Comment:
Why did you make this change in this PR?

Reply:
We need the constructors of QueryProcessors to conform with what yaml loading returns and this change was required to make the MappedGranularityProcessor constructor conform to a more primitive input time (`Mapping[int,int]`)